### ISSUE TITLE
Canon EOS 7D "Could not claim USB device" when i try to use as webcam.

### ISSUE THREAD

**Issue by RCBoater21** [gphoto/gphoto2#406]
I am on linux mint Ulyssa running on a Lenovo T410. Whenever i try to run anything involving the camera like "gphoto2 --capture-image-and-download", I just get "*** Error ***              
An error occurred in the io-library ('Could not claim the USB device'): Could not claim interface 0 (Device or resource busy). Make sure no other program (gvfs-gphoto2-volume-monitor) or kernel module (such as sdc2xx, stv680, spca50x) is using the device and you have read/write access to the device."

**Comment by msmeissn** [comment]
yes, it says as it is, the camera is blocked by anothe program, likel gvfs (which provides a virtual filesystem for nautilus).

gio mount -s gphoto2

should stop this kind of mounting temporary.
